backend/src/libs/interfaces/service.interface.ts

```diff
@@ -1,6 +1,6 @@
 interface IService<T = unknown> {
   find(id: unknown): Promise<T>;
-  findAll(): Promise<{
+  findAll(user: unknown): Promise<{
```

Comment:
```suggestion
  findAll(): Promise<{
```
add new method `findAllByUserId`